app/views/articles/_single_story.html.erb

```diff
@@ -14,7 +14,7 @@
   <% end %>
   <a href="/<%= story.cached_user.username %>" class="small-pic-link-wrapper">
     <div class="small-pic">
-      <img src="<%= ProfileImage.new(story.cached_user).get(width: 90) %>" alt="<%= story.cached_user.username %> profile" />
+      <img src="<%= ProfileImage.new(story.cached_user).get(width: 90) %>" alt="<%= story.cached_user.username %> profile" width="40" height="40" />
```

Comment:
I was wondering, but maybe you chose this because the height would be too short, if the width of the `<img` tag is 40, why not also ask for a `width: 40` in the method? Could you try and see what happens?

`ProfileImage.new(story.cached_user).get(width: 40)`

Reply:
But I actually want to have twice as big image so it looks sharp and crispy on retina when resized manually to 40x40.